daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -122,7 +145,7 @@ metrics:
     excludeVerbs: false
 ```
 
-In the examples above this path filter `/orders/{orderID}/items/{itemID}` would return a single metric count matching all the orderIDs and all the itemIDs rather than multiple metrics for each itemID. For more information see [HTTP metrics path matching]({{< ref "metrics-overview.md#http-metrics-path-matching" >}})
+In the examples above, the path filter `/orders/{orderID}/items/{itemID}` would return _a single metric count_ matching all the `orderID`s and all the `itemID`s, rather than multiple metrics for each `itemID`. For more information, see [HTTP metrics path matching]({{< ref "metrics-overview.md#http-metrics-path-matching" >}})
```

Comment:
The metrics link here is broken. It should point to For more information see [HTTP metrics path matching](https://docs.dapr.io/operations/observability/metrics/metrics-overview/#http-metrics-path-matching). Please try this out